lib/web/cookies/index.js

```diff
@@ -5,9 +5,7 @@ const { stringify } = require('./util')
 const { webidl } = require('../fetch/webidl')
 const { Headers } = require('../fetch/headers')
 
-const headersToBrandCheck = globalThis.Headers
-  ? [Headers, globalThis.Headers]
-  : [Headers]
+const brandChecks = webidl.brandCheckMultiple([Headers, globalThis.Headers].filter(Boolean))
```

Comment:
What if you run node with --no-experimental-fetch? Then globalThis.Headers will be undefined. Thus breaking the code.

Reply:
that's what the .filter(Boolean) is for

Reply:
Ah right